core/rawdb/database.go

```diff
@@ -256,12 +274,12 @@ func NewLevelDBDatabase(file string, cache int, handles int, namespace string, r
 
 // NewLevelDBDatabaseWithFreezer creates a persistent key-value database with a
 // freezer moving immutable chain segments into cold storage.
-func NewLevelDBDatabaseWithFreezer(file string, cache int, handles int, freezer string, namespace string, readonly bool) (ethdb.Database, error) {
+func NewLevelDBDatabaseWithFreezer(file string, cache int, handles int, freezer string, namespace string, readonly, isPruneBlock bool) (ethdb.Database, error) {
```

Comment:
This is a `rawdb`, it is not proper to see any wording like `isPruneBlock`

Reply:
Done